aws/resource_aws_dynamodb_table_test.go

```diff
@@ -927,6 +958,57 @@ resource "aws_dynamodb_table" "basic-dynamodb-table" {
 `, rName)
 }
 
+func testAccAWSDynamoDbConfigInitialStateWithEncryption(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
+  name = "%s"
+  read_capacity = 10
```

Comment:
Can you please set the test capacities to `1`? 💸

Reply:
Will do

Reply:
Will do